thirdweb/sdk.py

```diff
@@ -127,18 +128,29 @@ def get_pack_module(self, address: str) -> PackModule:
         return self.__pack_module
 
     @set_default_account
-    def get_market_module(self, address: str) -> MarketModule:
+    def get_market_module(self, address: str, isv1: bool = True) -> MarketModule:
```

Comment:
the deprecation message is wrong here - should direct ppl to use `get_marketplace_module`